Makefile

```diff
@@ -23,7 +24,7 @@ jenkins: .info build-jar build-linux push-linux docker-multi-arch clean
 #####
 build-jar: .info
 	mvn -f proxy --batch-mode package 
-	cp proxy/target/proxy-${VERSION}-uber.jar ${out}
+	cp proxy/target/${ARTIFACT_ID}-${VERSION}-uber.jar ${out}
```

Comment:
better to have the source of truth from pom.xml rather than have it hardcoded in Makefile.
Since we changed our artifactId from `wavefront` to `proxy` in pom.xml, all jenkins jobs need to be changed. Trying to avoid this if we change anything in the future. Better to maintain the code. Let me know what you think.